From 88b202cac591ed911d23ace32ad34350261bfdc7 Mon Sep 17 00:00:00 2001
From: ajayk <ajaykemparaj@gmail.com>
Date: Thu, 21 Sep 2023 10:30:36 -0700
Subject: [PATCH] webrick: we have latest version of webrick > 1.8.1

---
 fluentd.gemspec | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fluentd.gemspec b/fluentd.gemspec
index 43e8ae07..a0f10fd3 100644
--- a/fluentd.gemspec
+++ b/fluentd.gemspec
@@ -28,7 +28,7 @@ Gem::Specification.new do |gem|
   gem.add_runtime_dependency("tzinfo", [">= 1.0", "< 3.0"])
   gem.add_runtime_dependency("tzinfo-data", ["~> 1.0"])
   gem.add_runtime_dependency("strptime", [">= 0.2.4", "< 1.0.0"])
-  gem.add_runtime_dependency("webrick", [">= 1.4.2", "< 1.8.0"])
+  gem.add_runtime_dependency("webrick", [">= 1.4.2"])
 
   # build gem for a certain platform. see also Rakefile
   fake_platform = ENV['GEM_BUILD_FAKE_PLATFORM'].to_s
-- 
2.39.3 (Apple Git-145)

